privacy-config/privacy-config-api/src/main/java/com/duckduckgo/privacy/config/api/PrivacyConfigDownloader.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.privacy.config.api
+
+/** Public interface for download remote privacy config */
+interface PrivacyConfigDownloader {
```

Comment:
I don't think we should expose this method anywhere. `PrivacyConfigDownloader` should remain an implementation detail.

Reply:
Discussed offline, I will remove it and do it through a downloader plugin callback instead